plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotExpressionRewriter.java

```diff
@@ -156,6 +158,13 @@ private static <T> Optional<T> applyRule(RewriteRule<T> rule, T object, Context
         return Optional.empty();
     }
 
+    private static Literal toLiteral(String value)
+    {
+        Literal literal = new Literal();
+        literal.setStringValue(value);
+        return literal;
```

Comment:
Why not `Literal.stringValue()` method?

Reply:
It is overridden and surrounds non-null values with single quotes and null with the string `null`. Adding a comment to explain.

Reply:
Thanks! Updating.

Reply:
I mean the new `toLiteral` method seem the same as `org.apache.pinot.common.request.Literal#stringValue`. 
```java
  public static Literal stringValue(java.lang.String value) {
    Literal x = new Literal();
    x.setStringValue(value);
    return x;
  }
```

https://github.com/apache/pinot/blob/d9c4315ca14997e6e0300d04f788e07723875159/pinot-common/src/main/java/org/apache/pinot/common/request/Literal.java#L204-L208

Reply:
Yep, just updated it, thanks for that!